example/server.js

```diff
@@ -5,6 +5,6 @@ webpack.listen(8080, "localhost", function() {
   console.log("Server is listening on http://localhost:8080")
 });
 
-api.listen(7000, function() {
-  console.log('API listening at http://localhost:7000');
+api.listen(7654, function() {
```

Comment:
so that it's always 8081? Or are you saying make 8080 the port variable by default?